requirements.txt

```diff
@@ -1,5 +1,5 @@
 # Use >= for development versions so that source builds always work
-coala>=0.8.1
+coala>=0.9.0.dev20160915154755
```

Comment:
Don't forget to update this once https://github.com/coala-analyzer/coala/pull/2762 is merged!